repos/system_upgrade/el7toel8/actors/createresumeservice/actor.py

```diff
@@ -21,19 +22,26 @@ class CreateSystemdResumeService(Actor):
 
     def process(self):
         service_name = 'leapp_resume.service'
+        systemd_dir = '/etc/systemd/system'
+
         service_templ_fpath = self.get_file_path(service_name)
-        shutil.copyfile(service_templ_fpath, os.path.join(
-            '/etc/systemd/system/', service_name))
+        shutil.copyfile(service_templ_fpath, os.path.join(systemd_dir, service_name))
 
         service_path = '/etc/systemd/system/{}'.format(service_name)
         symlink_path = '/etc/systemd/system/default.target.wants/{}'.format(service_name)
 
+        # in case nothing is enabled in the default target, the directory does not exist
+        try:
+            os.mkdir(os.path.join(systemd_dir, 'default.target.wants'))
+        except OSError:
+            pass
```

Comment:
Maybe some log message?

Reply:
I could maybe do something like `if e.errno != errno.EEXIST:` and then some log might be useful but dunno, if this fails, there is probably much bigger problem...

Reply:
then raise StopActorExecution:)

Reply:
@pirat89 wdyt?

Reply:
I guess it's not so blocking, I would put there just the debug message as it's really just hypothetic error. something like check whether the path exists or not. if exists, continue. if not, .. depends on the rest of the code.

Reply:
It is related to # #520 anyway. I just used it here for convenience...

Reply:
I know, I wrote that it's more just only hypothetical issues. I think it's ok like that.

Reply:
I am leaving it up to you.